backend/python/app/services/implementations/auth_service.py

```diff
@@ -69,7 +69,7 @@ def generate_oauth_token(self, id_token):
                 CreateUserWithGoogleDTO(
                     first_name=google_user["firstName"],
                     last_name=google_user["lastName"],
-                    role="User",
+                    role_id="User",
```

Comment:
nit: this seems like a mistake, can you revert this line back to `role="User",`?